File: LR_3-4/блок 4/pack_4_zadanie_7.py
n=int(input('введите количество судей: '))
s = 0
min = 10
max = 0
for i in range(n):
    x = int(input('введите оценки судей: '))
    s += x
    if x > max:
        max = x
    if x < min:
        min = x
s = s - min - max
print(f'{s/(n-2):.1f}')

File: LR_3-4/блок 4/pack_4_zadanie_7.pas
program Zadanie_7;
uses CRT;
var n,i,s,min,max,x: integer;
begin
  writeln('введите количество судей');
  readln(n);
  s:=0;
  min:=10;
  max:=0;
  for i:= 1 to n do
  begin
    writeln('введите оценки судей');
    readln(x);
    s:=s+x;
    if x>max then max:=x;
    if x<min then min:=x;
  end;
  s:=s-min-max;
  writeln(s/(n-2):0:1);
end.